src/components/avatar/README.md

```diff
@@ -0,0 +1,59 @@
+# Avatar
+A text or image-based graphic to represent a person or user
+
+## When to use
+Use the avatar component to visually represent a person within content or a user interface. This could include:
+
+- A photo avatar of a person who said a quote (such as when used within the `quote` component)
```

Comment:
A photo of someone paired with relevant text (such as within the `quote` component)